dev_guide/new_app.adoc

```diff
@@ -437,6 +442,50 @@ $ oc new-app \
 ----
 ====
 
+// NB: See note at the end of "Output Without Creation".
+[[useful-edits]]
+==== Useful Edits
+
+Following are some specific examples of useful
+xref:output-without-creation[edits to make]
+in the *_myapp.yaml_* file.
+
+NOTE: These examples presume *_myapp.yaml_* was created as
+a result of the `oc new-app ... -o yaml` command.
+
+[[new-app-deploy-selected-nodes]]
+.Deploy to Selected Nodes
+====
+----
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Project  <1>
+  metadata:
+    name: myapp
+    annotations:
+      openshift.io/node-selector: region=west  <2>
+- apiVersion: v1
+  kind: ImageStream
+  ...
+kind: List
+metadata: {}
+----
+<1> In *_myapp.yaml_*, the section that defines the *myapp* project
+    has both `kind: Project` and `metadata.name: myapp`.
+    If this section is missing, you should add it at the *second* level,
+    as a new item of the list `items`,
+    peer to the `kind: ImageStream` definitions.
+<2> Add this
+    xref:../admin_guide/managing_projects.html#using-node-selectors[node selector]
```

Comment:
@tnguyen-rh I came across this adoc/html snafu while fixing some of my own, and repaired here:

https://github.com/openshift/openshift-docs/pull/2529/files#diff-e3f06b2c6d8c3c82ac37800dc85d217f

However I should have done them in separate PRs/commits cuz I needed mine on the enterprise-3.2 branch, yet your PR is still in Next Release.

Soooo, just highlighting this here in case it comes up during picking the next release (cc @bfallonf)